lib/reply.js

```diff
@@ -126,6 +126,7 @@ Reply.prototype.send = function (payload) {
   }
 
   if (payload instanceof Error || this[kReplyIsError] === true) {
+    this[kReplyIsError] = false
```

Comment:
Why this change? I'm a bit lost since `onErrorHook` and the entire workflow don't manipulate `kReplyIsError`

Reply:
The reason lines in further changes in this file. You already had to turn it to false at two different places. The kReplyIsError is checked only in this section, so it makes sense to me to reset it here instead of chasing it down across the code.

Even